src/vma/dev/cq_mgr.cpp

```diff
@@ -168,8 +168,8 @@ inline uint32_t cq_mgr::process_recv_queue(void* pv_fd_ready_array)
 }
 
 cq_mgr::cq_mgr(ring_simple* p_ring, ib_ctx_handler* p_ib_ctx_handler, int cq_size, struct ibv_comp_channel* p_comp_event_channel, bool is_rx) :
-		m_p_ring(p_ring), m_p_ib_ctx_handler(p_ib_ctx_handler), m_b_is_rx(is_rx), m_b_is_rx_sw_csum_on(safe_mce_sys().rx_sw_csum),
-		m_comp_event_channel(p_comp_event_channel), m_p_next_rx_desc_poll(NULL)
+		m_p_ring(p_ring), m_p_ib_ctx_handler(p_ib_ctx_handler), m_p_ib_ctx_time_converter(p_ib_ctx_handler->get_ctx_time_converter()),
```

Comment:
check if `m_p_ib_ctx_time_converter` need to be update at ring_restart(), after a bond HA even.
a new device + port might be assigned to this ring and it's HW time source will change with it.